src/service/inputs/parsed-inputs.ts

```diff
@@ -0,0 +1,25 @@
+import { defaultInputValues, InputValues, LoggerLevel } from "@bc/domain/inputs";
+import { Service } from "typedi";
+
+@Service()
+export class ParsedInputs {
+    // store parsed options
+    private _inputs: InputValues = defaultInputValues;
+
+    /**
+     * Getter for parsed options
+     * @returns Object containing parsed options
+     */
+    get inputs(): InputValues {
+        return this._inputs;
+    }
+
+    /**
+     * Updates the parsed options
+     * @param updatedInputs Options that were obtained from the parser
+     */
+    updateInputs(updatedInputs: InputValues) {
```

Comment:
this method is not covered by tests and it's really important one, don't think so?

Reply:
I see a 100% coverage for this file though
```
service/inputs                    |     100 |      100 |     100 |     100 |                   
   input-service.ts               |     100 |      100 |     100 |     100 |   
```